src/style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  background-color: rgb(255, 255, 255);
+}
+
+.list-container {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #ccc;
+  margin: 40px;
+}
+
+i {
+  position: absolute;
+  right: 30px;
+  padding: 5px;
+}
+
+h1 {
+  font-size: 18px;
+}
+
+a {
+  color: black;
+}
+
+.addRow {
+  padding: 0;
+}
+
+.addList {
+  width: 100%;
+  padding: 20px 20px;
+  margin: 0;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.addRow i {
+  position: absolute;
+  right: 30px !important;
+  bottom: 10px !important;
```

Comment:
It is a better practice avoid to use the `!important` keyword. It goes against the normal flow of CSS and it could produce unexpected behaviors.

Reply:
Yes sure I will not use more just here